### ISSUE TITLE
moment.locale does not work after 2.15.0

### ISSUE THREAD

**Issue by vansosnin** [moment/moment#3439]
`moment.locale("ru")` does not set right locale.

Reproduce:

```javascript
import moment from "moment";
console.log(moment());
moment.locale("ru");
console.log(moment());
```

In both console.logs I see `_locale._abbr === "en"`.

I debugged a little and noticed that [here](https://github.com/moment/moment/commit/c587bf0063c18214238ca83f3f9c6170daf25bed#diff-e5443d712ed6c7ad75397f60b06a1d84R1794) I have no `module.require`. And `locales[name]` few lines after obviously returns _undefined_.

**Comment by Hadrien-DELAITRE** [comment]
Same bug for me

`module.require` is `undefined` browser-side, so locale is not setted.

**Comment by icambron** [comment]
See fix in #3438, will be release as 2.15.1
